api/v1alpha1/gatekeeper_types.go

```diff
@@ -43,205 +43,289 @@ func (m Mode) ToBoolString() string {
 	Operations []OperationType `json:"operations,omitempty"`
+
+	// DisabledBuiltins is a list of specific OPA built-in functions to disable. By default, http.send
+	// is disabled.
+	// See https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#disable-opa-built-in-functions.
+	//
 	// +optional
 	DisabledBuiltins []string `json:"disabledBuiltins,omitempty"`
+
+	// LogMutations enables logging of mutation events and errors. The default value is Disabled.
+	// See https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#beta-enable-mutation-logging-and-annotations.
+	//
 	// +optional
-	// Sets the --log-mutations flag which enables logging of mutation events and errors. This defaults to Disabled.
+	//nolint:lll
 	LogMutations *Mode `json:"logMutations,omitempty"`
+
+	// MutationAnnotations adds the gatekeeper.sh/mutation-id and gatekeeper.sh/mutations annotations
+	// to mutated objects. The default value is Disabled.
+	// See https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#beta-enable-mutation-logging-and-annotations.
+	//
 	// +optional
-	// Sets the --mutation-annotations flag which adds the gatekeeper.sh/mutation-id and gatekeeper.sh/mutations
-	// annotations on mutated objects. This defaults to Disabled.
+	//nolint:lll
 	MutationAnnotations *Mode `json:"mutationAnnotations,omitempty"`
+
+	// Replicas is the number of desired pods for the admission webhook.
+	//
+	// +optional
+	// +kubebuilder:validation:Minimum:=0
+	Replicas *int32 `json:"replicas,omitempty"`
+
+	// Resources describes the compute resource requirements for the admission webhook pods.
+	//
+	// +optional
+	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// +optional
+	LogLevel *LogLevelMode `json:"logLevel,omitempty"`
 }
 
-// +kubebuilder:validation:Enum:=CONNECT;CREATE;UPDATE;DELETE;*
-type OperationType admregv1.OperationType
+type ImageConfig struct {
+	// Deprecated: Image is deprecated. Its continued use will be honored by the operator with a
+	// warning and removed in a future release. Instead, the operator will rely on the environment
+	// variable set in its manifest at deployment time and will be the default behavior after this
+	// field is removed.
+	// Image to pull including registry (optional), repository, name, and tag
+	// e.g. quay.io/gatekeeper/gatekeeper-operator:latest
+	//
+	// +optional
+	Image *string `json:"image,omitempty"`
 
-// +kubebuilder:validation:Enum:=DEBUG;INFO;WARNING;ERROR
-type LogLevelMode string
+	// +optional
+	ImagePullPolicy *corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+}
 
-const (
-	LogLevelDEBUG   LogLevelMode = "DEBUG"
-	LogLevelInfo    LogLevelMode = "INFO"
-	LogLevelWarning LogLevelMode = "WARNING"
-	LogLevelError   LogLevelMode = "ERROR"
-)
+type ConfigConfig struct {
+	// Matches is a list of exempt namespaces for specified processes. If specified, the Matches will
+	// be appended to the existing Config spec.match.
```

Comment:
It doesn't actually append it. It just appends to the default matches.
```suggestion
	// be set on the existing Config spec.match.
```